### DISCUSSION TITLE
How to save a instance of the chart ?

### DISCUSSION THREAD

**Discussion by rdebb** [imaNNeo/fl_chart#1288] [Q&A]
How to save a instance of the chart ?

**Comment by imaNNeo** [comment]
What do you mean?

**Reply by rdebb** [reply]
I want to export the char visualisation as image or pdf like this

**Reply by imaNNeo** [reply]
Got it. 
You can take a screenshot from a widget.
I am pretty sure there are a lot of resources on the internet to check.
